specification/purview/data-plane/Azure.Analytics.Purview.DataPolicies/preview/2021-01-01-preview/purviewDataPolicy.json

```diff
@@ -0,0 +1,835 @@
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "values"
+        },
+        "x-ms-examples": {
+          "DataPolicyRoles_List": {
+            "$ref": "./examples/DataPolicyRoles_List.json"
+          }
+        }
+      }
+    },
+    "/dataPolicies": {
+      "get": {
+        "tags": [
+          "DataPolicy"
+        ],
+        "description": "List data policy",
+        "operationId": "DataPolicy_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DataPolicyList"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Policy Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "values"
+        },
+        "x-ms-examples": {
+          "DataPolicy_List": {
+            "$ref": "./examples/DataPolicy_List.json"
+          }
+        }
+      }
+    },
+    "/dataPolicies/{policyName}": {
+      "put": {
+        "tags": [
+          "DataPolicy"
+        ],
+        "description": "Updates a data policy",
+        "operationId": "DataPolicy_CreateOrUpdate",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "policyName",
+            "description": "Unique policy name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Policy object to be create or updated. .",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DataPolicy"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DataPolicy"
+            }
+          },
+          "200": {
```

Comment:
DataPolicyBinding_Create does not use a 200 response for updates, this one does. Is this intentional?

Reply:
Yes, DataPolicyBinding_Create is intended to only for creation and not updates.